roles/cinder-data/tasks/main.yml

```diff
@@ -78,7 +78,7 @@
   systemd_service: name=cinder-volume user=cinder
                    cmd=/usr/local/bin/cinder-volume
                    config_dirs=/etc/cinder
-  when: os == 'rhel' and openstack_install_method == 'package'
+  when: openstack_install_method != 'distro'
```

Comment:
this is configuring systemd service so has to be for rhel/centos. The reason this passed in CI is bcs centos openstack_install_method == source ..

What is expected behavior on openstack_install_method == 'distro' ? is openstack rpm install creating the service and thus we need to skip this?